tests/fixtures/ci/docker/debian.Dockerfile

```
@@ -9,36 +10,41 @@ FROM debian:bookworm-slim
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # === SECTION: base packages (managed) ===
-# Source: template (distro-specific package list)
+# Source: template (distro-specific seed contract)
+# The seed is the whole host contract: curl fetches mise, git is what uv shells
+# out to for the flext-infra git+https requirement, make invokes the verbs.
+# libicu-dev is pulled in because tokei (cargo-backed) needs a Rust toolchain,
+# which in turn needs it — mise provisions Rust, so the native
+# ICU headers must be present at the system layer.
 RUN apt-get update \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)